projects/manage-pom-cases-and-delius/src/main/resources/application.yml

```diff
@@ -61,7 +61,9 @@ seed.database: true
 wiremock.enabled: true
 context.initializer.classes: uk.gov.justice.digital.hmpps.wiremock.WireMockInitialiser
 
-messaging.consumer.queue: message-queue
+messaging:
+  consumer.queue: message-queue
+  producer.queue: message-queue
```

Comment:
I think      ` MESSAGING_PRODUCER_QUEUE: QUEUE_NAME` will need to be put in values.yaml

Reply:
done